### ISSUE TITLE
Browserslist: caniuse-lite is outdated

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#954]
When building the frontend I get:

```
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
```

**Comment by 4ydan** [comment]
Sorry totally missed this, done in #1032

**Comment by markus2330** [comment]
#1032 is an issue and not a PR?

**Comment by 4ydan** [comment]
Sry, I meant #1046

**Comment by markus2330** [comment]
Ok, the version pump there needs to be removed then so that we can merge this before the release. Or we simply skip 0.3.5 and directly release 0.3.6? It is only a number after all.

**Comment by 4ydan** [comment]
Done.

**Comment by 4ydan** [comment]
Is this still an issue?

**Comment by markus2330** [comment]
I think it was fixed by you in a6dca28fc225b25eb0825378889223c6278894ca

I close it for now. Eventually it will again to be updated `npm install` complaints then, then we can reopen this issue.
